src/Workspaces/Core/Portable/WorkspaceImmutableArrayExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.CodeAnalysis
+{
```

Comment:
The workspaces layer already has an ImmutableArrayExtensions class under the Roslyn.Utilities namespace: [https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Portable/Utilities/ImmutableArrayExtensions.cs](https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Portable/Utilities/ImmutableArrayExtensions.cs). Please either unify these extensions or move the extensions introduced here to that class, but don't put multiple ImmutableArrayExtensions classes in that project.

Reply:
Sweet, didn't see that.